benchmark.rb

```diff
@@ -22,13 +21,15 @@
 obj.delete(:'ActiveRecord')
 obj.delete(:Complex)
 
+Oj::Rails.optimize(Array, BigDecimal, Hash, Range, Regexp, Time, ActiveSupport::TimeWithZone, User)
```

Comment:
It is too hard. Users install `oj` and hope everything is done. It took me some time to realise and prove that `to_json` inside Rails project is slow. It is not a choice. It is simple lack of knowledge.